### DISCUSSION TITLE
Import Google Analytics 4 Data?

### DISCUSSION THREAD

**Discussion by octoxan** [plausible/analytics#3912] [Self-Hosted Support] [answered]
Is there any way to import data from Google Analytics 4 (GA4) or is only Google Universal Analytics supported currently? UA is being sunset in July 2024, and we're mostly trying Plausable for people who were unhappy with GA4 after having it set up for a few months already.

If it's not possible, do you foresee it being possible in the future?

Thanks!

**Comment by ruslandoga** [comment, marked as answer]
👋 @octoxan 

It's in the works: https://github.com/plausible/analytics/pulls?q=is%3Apr+ga4

**Reply by octoxan** [reply]
Marvelous! Sorry I had searched issues on the community plausible repo, and discussions and forgot to search open PRs over here 😅

**Reply by ruslandoga** [reply]
GA4 import is shipped in v2.1.0-rc.1: https://github.com/plausible/analytics/discussions/4084
